src/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const DEV_ENVRONMENT = process.env.DEV_ENVIRONMENT || null;
```

Comment:
minor) how about `NODE_ENV` ('dev' || 'prod' and defaults to undefined, meaning local)?

Reply:
+1 (treat it as a comcom convention for easy project management?)

also a typo: ..ENVIRON..

Reply:
@liayoo @platfowner Done! Thanks!